Soleras3/react-challenge-project-jan-2023
# Fix user email redux store

## Changes
- Fix the faulty pointer in the redux auth reducer from `payload.login` to `payload.email`.

## Purpose
User email wasn't being stored after login for use on other pages.

## Approach
Only one way to fix. See #2, same type of issue.

## Pre-Testing TODOs
Requires https://github.com/Soleras3/react-challenge-project-jan-2023/pull/2 to create orders.

## Testing Steps
1. Login and create an order.
2. Navigate to View Orders and observe the created order's "Created by" shows the logged in email.

## Learning
I'm not too familiar yet with Redux so this led me to learn about it-- specifically [reducers](https://redux.js.org/tutorials/fundamentals/part-2-concepts-data-flow)-- on redux.js.org.

Closes https://github.com/Shift3/react-challenge-project-jan-2023/issues/4
